src/test/resources/testscripts/DeleteWorkspace.sh

```diff
@@ -6,8 +6,7 @@ terra status
 terra auth status
 
 isLoggedIn=$(terra auth status --format=json | jq .loggedIn)
-if [[ "true" = "$isLoggedIn" ]]
-then
+if [[ "true" = "$isLoggedIn" ]]; then
```

Comment:
Could you also (opportunistically) replace single "=" with double "=="?

From https://google.github.io/styleguide/shellguide.html#testing-strings

_For clarity, use == for equality rather than = even though both work. The former encourages the use of [[ and the latter can be confused with an assignment._

Reply:
Done